test/cryptography/convert.js

```diff
@@ -260,7 +260,7 @@ describe('convert', () => {
 			const stringifiedEncryptedPassphrase =
 				'salt=e8c7dae4c893e458e0ebb8bff9a36d84&cipherText=c0fab123d83c386ffacef9a171b6e0e0e9d913e58b7972df8e5ef358afbc65f99c9a2b6fe7716f708166ed72f59f007d2f96a91f48f0428dd51d7c9962e0c6a5fc27ca0722038f1f2cf16333&iv=1a2206e426c714091b7e48f6&tag=3a9d9f9f9a92c9a58296b8df64820c15&version=1';
 			const encryptedPassphrase = {
-				iterations: null,
+				iterations: undefined,
```

Comment:
What should happen when iterations was non-number, which results in NaN?
ie) `iteration=` or `iteration=null`